libs/design-system/filter-container/src/lib/filter-container.component.ts

```diff
@@ -0,0 +1,69 @@
+import { booleanAttribute, ChangeDetectionStrategy, Component, input, model } from '@angular/core';
+import { MatChipsModule } from '@angular/material/chips';
+import { MatIconModule } from '@angular/material/icon';
+import { MatDividerModule } from '@angular/material/divider';
+import { HraCommonModule } from '@hra-ui/common';
+import { ButtonsModule } from '@hra-ui/design-system/buttons';
+import { RichTooltipModule } from '@hra-ui/design-system/tooltips/rich-tooltip';
+
+/** A filter chip representing a selected filter option */
+export interface FilterChip {
+  /** Unique identifier for the chip */
+  id: string;
+  /** Display label for the chip */
+  label: string;
+  /** Optional secondary label */
+  secondaryLabel?: string;
+}
+
+/** Configuration for the rich tooltip info button */
+export interface RichTooltipConfig {
+  /** Tooltip description */
+  description: string;
+  /** Optional action text for the tooltip */
+  actionText?: string;
+  /** Optional action URL for the tooltip */
+  actionUrl?: string;
+}
+
+/**
+ * Design system filter container component
+ */
+@Component({
+  selector: 'hra-filter-container',
+  imports: [HraCommonModule, ButtonsModule, RichTooltipModule, MatIconModule, MatChipsModule, MatDividerModule],
+  templateUrl: './filter-container.component.html',
+  styleUrl: './filter-container.component.scss',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class FilterContainerComponent {
+  /** tagline for the filter category */
+  readonly tagline = input.required<string>();
```

Comment:
Sorry, I've misunderstood what this property was. Could you please rename it to `action`?